config/deploy.rb

```diff
@@ -301,7 +301,7 @@
   date = Time.now.strftime("%Y-%m-%d")
   file_name = "backup-#{date}-#{fetch(:stage)}"
   on release_roles :db do
-    execute "sudo -u root -i mysqldump #{ fetch(:db_name) } > /tmp/#{file_name}.sql"
+    execute "#{ fetch(:db_name) } > /tmp/#{file_name}.sql"
```

Comment:
@kevinreiss do you need mysqldump in line 304?

Reply:
yes added back